app/models/passenger.rb

```diff
@@ -3,6 +3,8 @@
 require 'csv'
 
 class Passenger < ApplicationRecord
+  attr_writer :needs_longer_rides
+
```

Comment:
```suggestion
```
Remember we talked about this:

https://github.com/umts/st-pax/blob/324526a2af2eacb5be376edee8db2621a8d8971a/app/models/passenger.rb#L44-L46

Namely that `needs_longer_rides` is not an attribute on `Passenger`. It's just a method that checks if the passenger has a `MobilityDevice` that has `mobility_device.needs_longer_rides == true`.

As such, we shouldn't be adding anything to the model, you just need to set up your test data correctly - more on that below